project/src/components/host/host.tsx

```diff
@@ -0,0 +1,35 @@
+import { CardProps } from '../../types/offer';
+
+type Props = {
+  card: CardProps;
```

Comment:
```suggestion
  card: CardProps;
```

CardProps лучше Card